express/features/support/steps/shared-functions.ts

```diff
@@ -95,3 +95,16 @@ function getSingleElement(elements: ElementHandle[], match: string): ElementHand
     assert.equal(elements.length, 1, `Not exactly one element matched ${match} (matches: ${elements.length})`);
     return elements[0];
 }
+
+export async function getElementById(page: Page, elementId: string): Promise<ElementHandle> {
+    const element = await page.$(`#${elementId}`);
+    if (!element) {
+        throw new Error(`Could not find element with id ${elementId}`);
+    }
+    return element;
+}
```

Comment:
This function is not really needed. The `#${elementId}` selector can be used in puppeteer functions and if it doesn't find the element with the specified id it will throw the same error.